ci-scripts/yocto.groovy

```diff
@@ -255,7 +256,8 @@ void buildManifest(String variantName, String imageName, String layerToReplace="
 
         // Build the images
         try {
-            boolean buildUpdate = variantName.startsWith("rpi")
+            boolean buildUpdate = variantName.startsWith("rpi") || variantName.startsWith("intel") || variantName.startsWith("arp")
+            echo "testing for arp building"
```

Comment:
remove this line. This seemed to be added for debugging purpose.

Reply:
Done